rules/pulpos/default_rules.mk

```diff
@@ -28,6 +28,8 @@ endif
 
 ifdef gui
 override runner_args += --config-opt=**/vsim/gui=true
+else
+vsim-flags = -c
```

Comment:
This should already be added in the default flow, no? I see that it is needed in the custom flow for the cluster, but...

Reply:
[runner_args](https://github.com/pulp-platform/pulp-runtime/blob/504164678b5259a79c40b3f78d2fc61a01504d09/rules/pulpos/default_rules.mk#L276) is currently used only when the target platform is `gvsoc`. I can work only with `runner_args` and define it according to the target, what do you think?

Reply:
see above!